test/Polly.Extensions.Tests/Polly.Extensions.Tests.csproj

```diff
@@ -16,7 +16,10 @@
   <ItemGroup>
     <Using Include="Polly.TestUtils" />
     <ProjectReference Include="..\..\src\Polly.Extensions\Polly.Extensions.csproj" />
+    <ProjectReference Include="..\..\src\Polly.RateLimiting\Polly.RateLimiting.csproj" />
     <ProjectReference Include="..\Polly.TestUtils\Polly.TestUtils.csproj" />
+    <PackageReference Include="Microsoft.AspNetCore.Http" />
+    <PackageReference Include="Microsoft.AspNetCore.Http.Abstractions" />
     <PackageReference Include="Microsoft.Extensions.Configuration" />
     <PackageReference Include="Microsoft.Extensions.Options.ConfigurationExtensions" />
```

Comment:
```suggestion
    <ProjectReference Include="..\..\src\Polly.Extensions\Polly.Extensions.csproj" />
    <ProjectReference Include="..\..\src\Polly.RateLimiting\Polly.RateLimiting.csproj" />
    <FrameworkReference Include="Microsoft.AspNetCore.App" />
    <PackageReference Include="Microsoft.Extensions.Configuration" />
    <PackageReference Include="Microsoft.Extensions.Options.ConfigurationExtensions" />
```

Reply:
Not relevant as no new dependencies were added.